src/pages/PicnicQuest.vue

```diff
@@ -1,5 +1,10 @@
 <template>
```

Comment:
Compact picnic quest is on the backburner I suppose? It's NBD for the release

Reply:
yes absolutely, it's a secondary concern